code/modules/mob/living/carbon/human/species/station/tajara/tajara.dm

```diff
@@ -44,7 +44,7 @@
 	ethanol_resistance = 0.8//Gets drunk a little faster
 	rarity_value = 2
 	economic_modifier = 7
-	selectable_pronouns = null
+	selectable_pronouns = list(MALE, FEMALE)
```

Comment:
Wait I thought it was established that enbies were a thing, even if they weren't acknowledged by Taj communities? If a Taj told whoever makes the records in the SCC "Hi, they do not identify as male orr female", I'm fairly sure they'd write that down with no fuss.

Reply:
Pronouns in game are more than a formality in the records/id. As you can also see by the pr, the biological sex depends on the pronouns. Finally, as I explained in the discord, it will likely come in a future update.